apps/widget-e2e/src/util/Table.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export function selectRow(
+  columnName: string,
+  columnValue: string,
+  srPrefix?: string
+): void {
+  const selectorPrefix = srPrefix ? `${srPrefix}` : "";
+  const columnsHeaderColumn = `${selectorPrefix} [data-automationid='ColumnsHeaderColumn']`;
```

Comment:
is this data-automationid from office fabric?
I am not seeing it is defined anywhere in our code.

Reply:
Yes, this is from office fabric